# HG changeset patch
# User Rik <rik@octave.org>
# Date 1411273205 25200
#      Sat Sep 20 21:20:05 2014 -0700
# Node ID 3cb64f94294ff832b60253b465c8bcc0cd7cd7da
# Parent  3fb075f5140f7fd51c4af78bda3d4043442178e7
build-sparse-tests.sh: Use isfinite rather than deprecated finite.

* test/build-sparse-tests.sh: Use isfinite rather than deprecated finite.

diff --git a/test/build-sparse-tests.sh b/test/build-sparse-tests.sh
--- a/test/build-sparse-tests.sh
+++ b/test/build-sparse-tests.sh
@@ -459,17 +459,17 @@ print_mapper_test asin
 print_mapper_test asinh
 print_mapper_test atan
 print_mapper_test atanh
 print_mapper_test ceil
 print_mapper_test conj
 print_mapper_test cos
 print_mapper_test cosh
 print_mapper_test exp
-print_mapper_test finite
+print_mapper_test isfinite
 print_mapper_test fix
 print_mapper_test floor
 print_mapper_test imag
 print_mapper_test isinf
 print_mapper_test isna
 print_mapper_test isnan
 print_mapper_test log
 #print_mapper_test log10   ## fails with different NaN, not a problem
